src/pages/resources.tsx

```diff
@@ -0,0 +1,162 @@
+    downloads: [
+      {
+        fileName: "TradeTrust Tech Webinar 4.pdf",
+        path: "/static/images/webinar/tradetrust-tech-webinar-4.pdf",
+      },
+    ],
+  },
+  {
+    title: "Set Up and Change Templates",
+    youtubeEmbedCode: "z3l9OSVGHH8",
+    tag: "Technical",
+    description:
+      "Building on sessions 2 and 3, this session will walk you through the steps on how to: configure documents; set up schema in a configuration file; and change document templates.",
+    downloads: [
+      {
+        fileName: "TradeTrust Tech Webinar 5.pdf",
+        path: "/static/images/webinar/tradetrust-tech-webinar-5.pdf",
+      },
+    ],
+  },
+  {
+    title: "Identity Resolution Services",
+    youtubeEmbedCode: "3S1QjgPXn54",
+    tag: "Technical",
+    description:
+      "Identity Resolution in TradeTrust involves the ability to map non-readable identification of issuers to a readable form. This segment will cover the methods and the steps on building APIs for identity resolution.",
+    downloads: [
+      {
+        fileName: "TradeTrust Tech Webinar 6.pdf",
+        path: "/static/images/webinar/tradetrust-tech-webinar-6.pdf",
+      },
+    ],
+  },
+];
+
+export const ResourcesPage = () => (
+  <>
+    <Helmet>
+      <meta
+        property="description"
+        content="This series of tech talks is organised by the Infocomm Media Development Authority of Singapore (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with knowledge on TradeTrust as a digital utility for cross border trade."
+      />
+      <meta
+        property="og:description"
+        content="This series of tech talks is organised by the Infocomm Media Development Authority of Singapore (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with knowledge on TradeTrust as a digital utility for cross border trade."
+      />
+      <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+      <meta property="og:url" content={`${window.location.origin}/resources`} />
+      <title>TradeTrust - Resources</title>
+    </Helmet>
+
+    <div className="bg-blue-lighter">
+      <div className="container-custom py-5">
+        <div className="row">
+          <div className="col-12">
+            <h1>Resources</h1>
+          </div>
+        </div>
+        <div className="row py-3">
+          <div className="col-12 col-md-12 col-lg-8">
+            {resources.map((resource, index) => (
+              <ResourcesCard details={resource} key={index} />
+            ))}
+          </div>
+          <div className="col-12 col-md-12 col-lg-4">
+            {sideLinks.map((resourceLink, index) => (
+              <ResourcesLink
+                key={index}
+                title={resourceLink.title}
+                type={resourceLink.type}
+                details={resourceLink.details}
+              />
+            ))}
```

Comment:
the resource links column wanna put first followed by the resource cards column? then in `lg` upwards can do the `order-lg-1` and `order-lg-2` to flip them accordingly

this way in mobile your the resource links column will appear first, looks better imo

Reply:
ok